frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx

```diff
@@ -44,26 +46,65 @@ const SGID_SUPPORTED: Set<MyInfoAttribute> = new Set([
   // phone number formats.
   // MyInfo phone numbers support country code, while sgID-MyInfo does not.
   // MyInfoAttribute.MobileNo,
-
-  // FRM-1189: This is disabled due to slight different formatting.
-  // We format the Myinfo response by separates lines in addresses with comma
-  // Whereas sgID separates each line with newline.
-  // This should be enabled in future work
-  // MyInfoAttribute.RegisteredAddress,
-])
-
-/**
- * If sgID is used, checks if the corresponding
- * MyInfo field is supported by sgID.
- */
-const sgIDUnSupported = (
-  form: AdminFormDto | undefined,
-  fieldType: MyInfoAttribute,
-): boolean =>
-  form?.authType === FormAuthType.SGID_MyInfo && !SGID_SUPPORTED.has(fieldType)
+]
+const SGID_SUPPORTED_V2 = [
+  MyInfoAttribute.Sex,
+  MyInfoAttribute.Race,
+  MyInfoAttribute.Nationality,
+  MyInfoAttribute.HousingType,
+  MyInfoAttribute.HdbType,
+  MyInfoAttribute.RegisteredAddress,
+  MyInfoAttribute.BirthCountry,
+  MyInfoAttribute.VehicleNo,
+  MyInfoAttribute.Employment,
+  MyInfoAttribute.WorkpassStatus,
+  MyInfoAttribute.Marital,
+  MyInfoAttribute.MobileNo,
+]
 
 export const MyInfoFieldPanel = () => {
   const { data: form, isLoading } = useCreateTabForm()
+
+  const { user } = useUser()
+
+  // FRM-1444: Remove once rollout is 100% and stable
+  const growthbook = useGrowthBook()
+
+  useEffect(() => {
+    if (growthbook) {
+      growthbook.setAttributes({
+        // Only update the `adminEmail` and `adminAgency` attributes, keep the rest the same
+        ...growthbook.getAttributes(),
+        adminEmail: user?.email,
+        adminAgency: user?.agency.shortName,
+      })
+    }
+  }, [growthbook, user])
+
+  const showNewSgidMyInfoFields = useFeatureIsOn(featureFlags.myinfoSgid)
+
+  const SGID_SUPPORTED_FINAL = useMemo(() => {
+    return showNewSgidMyInfoFields
+      ? SGID_SUPPORTED_V1.concat(SGID_SUPPORTED_V2)
```

Comment:
Why not just make SGID_SUPPORTED_V2 include SGID_SUPPORTED_V1 since it makes more sense that way?

i.e.
```
const SGID_SUPPORTED_V2 = [
  MyInfoAttribute.Sex,
  MyInfoAttribute.Race,
  MyInfoAttribute.Nationality,
  MyInfoAttribute.HousingType,
  MyInfoAttribute.HdbType,
  MyInfoAttribute.RegisteredAddress,
  MyInfoAttribute.BirthCountry,
  MyInfoAttribute.VehicleNo,
  MyInfoAttribute.Employment,
  MyInfoAttribute.WorkpassStatus,
  MyInfoAttribute.Marital,
  MyInfoAttribute.MobileNo,
  ...SGID_SUPPORTED_V1
]
```

Reply:
Yes good point, changed in [8f745d6](https://github.com/opengovsg/FormSG/pull/6766/commits/8f745d6c28751c2f52850d1bcfc8d2d276e53da2)